src/css-image-value.js

```diff
@@ -0,0 +1,39 @@
+(function(internal, scope) {
+
+  function CSSImageValue(image) {
+    if (!(image instanceof Image)) {
```

Comment:
Hmm.. I think that is for the CSSURLImageValue. Or should I just make this as a CSSURLImageValue? Actually I want to make something like this: https://github.com/css-typed-om/typed-om/compare/master...anthony-huang:try_CSSImageValue